content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md

```diff
@@ -42,10 +42,10 @@ table th:nth-of-type(2) {
 
 ## Resolved Issues
 
-| Topic  | Specific Change | 
+| Domain  | Specific Change | 
 | -------------------------------------------- | -------------------------------------------------- |
 | **CorDapp Development**                      | **Flows with Transient Errors** - Flows did not correctly handle transient errors while sessions were in progress. In some cases, the 'error' flag was cleared early by other events, incorrectly indicating that the error had been resolved. This flag can now only be cleared when the event that caused the transient error is retried.  |
 
 ## Known Limitations and Issues
 
-There is no support for upgrades from the early access beta versions.
+There is no support for upgrades from the beta versions.
```

Comment:
what does this mean? I think we should be saying "During the Beta process we do not guarantee the stability of our user APIs, thus seemless upgrade between Beta versions is not supported. 

After all, we know that its possiblem, only that we don't garuntee it

Reply:
Updated. Thanks.